corehq/ex-submodules/auditcare/models.py

```diff
@@ -155,7 +155,7 @@ def calculate_checksum(cls, instance_json, is_django=False):
             #if it's an existing version, then save it
             instance_copy.pop('_rev')
             json_string = json.dumps(instance_copy)
-        return hashlib.sha1(json_string).hexdigest()
+        return hashlib.sha1(json_string.encode('utf-8') if six.PY3 else json_string).hexdigest()
```

Comment:
what's the failure mode here if it can't encode? These are the HIPAA logs, so it's fairly high risk if the json from couch is not guaranteed to be utf-8

Reply:
For Python 3 (not Python 2): The result of `json.dumps` is Unicode, and therefore can always be encoded as UTF-8 unless you pass `ensure_ascii=False` (I'm not familiar with the edge cases there, so it may actually still be encodeable as UTF-8, even when passing `ensure_ascii=False`).

https://docs.python.org/3.6/library/json.html#character-encodings